src/components/PrimeIntro.vue

```diff
@@ -21,24 +22,24 @@
           <span v-else>You will get your domain along with</span>
         </div>
       </div>
-      <div class="modal-prime-intro__list">
-        <div class="modal-prime-intro__list__item">
+      <ul class="modal-prime-intro__list">
+        <li class="modal-prime-intro__list__item">
           <i class="icon-themes" /> Overlay, widget & site themes
-        </div>
-        <div class="modal-prime-intro__list__item">
+        </li>
+        <li class="modal-prime-intro__list__item">
           <i class="icon-store" /> FREE apps
-        </div>
-        <div class="modal-prime-intro__list__item">
+        </li>
+        <li class="modal-prime-intro__list__item">
           <i class="icon-upperwear" /> Custom merch store
-        </div>
-        <div class="modal-prime-intro__list__item">
+        </li>
+        <li class="modal-prime-intro__list__item">
           <i class="icon-loyalty" /> Gold status + FREE t-shirt
-        </div>
-        <div class="modal-prime-intro__list__item">
-          <i class="icon-phone" /> Stream on mobile
-        </div>
-        <span>...only from $12/month</span>
-      </div>
+        </li>
+        <li class="modal-prime-intro__list__item">
+          <i class="icon-phone-case" /> Stream on mobile
+        </li>
+        <span class="modal-prime-intro__list__price">...only from $12/month</span>
```

Comment:
`span` should be outside a `ul` since a `li` is only accepted directly under `ul`